docs/src/appendix/cast/script/script.md

```diff
@@ -0,0 +1,7 @@
+# `script`
+Compile and run a cairo deployment script or initialize script template by using subcommand [`init`](./init.md)
+
+## `<MODULE_NAME>`
+Required.
+
+Module name that contains the 'main' function that will be executed.
```

Comment:
Let's the argument name consistent with `init`

```suggestion
## `<SCRIPT_NAME>`
Required.

Script module name that contains the 'main' function that will be executed.
```

Reply:
Apologies these docs are confusing now. It is correct as it is now, because this is `MODULE_NAME` used to run script, not to init. I will fix it in next PR with `script run`.